editor/css/diagram.css

```diff
@@ -56,6 +56,12 @@ body {
 .executed .sprotty-node {
   fill: hsl(60 100% 50% / 50%);
 }
+
+.executed.last .sprotty-node {
+  stroke-width: 3px;
+  fill: hsl(60 100% 50% / 100%);
```

Comment:
How does this looks in dark mode?

Reply:
Brighter than simply executed:
![image](https://user-images.githubusercontent.com/93579455/170003112-2519baa3-5425-43ab-9f24-da2ebc33bdf7.png)

Reply:
![image](https://user-images.githubusercontent.com/93579455/170008687-e0d459df-861b-4102-91a4-de8bba74a702.png)
vs.
![image](https://user-images.githubusercontent.com/93579455/170008814-55c7403a-595e-45bb-95fb-51e526c8df1c.png)

Reply:
I don't like the white color inside the bright yellow! Very hard to read...

Reply:
I leave it open for QG, assuming that the animation color will be changed to green ;-)